orderpage.html

```diff
@@ -17,10 +19,7 @@ <h3>Choose your bread:</h3>
             <input type="radio" name="breads" value="Bagel"/> Bagel
             <input type="radio" name="breads" value="Teff Wrap"/> Teff Wrap without Honey
             <input type="radio" name="breads" value="English Muffin"/> English Muffin
-          </legend>
-        </fieldset>
-        <fieldset>
-          <legend>
+
             <h3>Select up to two spreads:</h3>
               <p>
```

Comment:
FYI this p tag isn't needed, the inputs can hang on their own, thanks boo

Reply:
I can't figure out how to get the css to work if i don't leave the p tag? I tried Input with the same properties and it didn't work?